include/flatbuffers/util.h

```diff
@@ -45,13 +50,63 @@
 
 namespace flatbuffers {
 
+#ifdef FLATBUFFERS_PREFER_PRINTF
+template<typename T> size_t IntToDigitCount(T t) {
+  size_t digit_count = 0;
+
+  // Count the sign for negative numbers
+  if (t < 0) {
+    digit_count++;
+  }
+
```

Comment:
remove blank lines from this and other functions. For very simple If's like this, make them one-liners.

Reply:
Done in 51e2098